.ci/shippable.sh

```diff
@@ -131,6 +131,15 @@ validate-ci-temp-empty)
   exit $fail
   ;;
 
+git-status)
+  if [ "$(git status --porcelain=v1 --untracked-files=normal)" ]; then
```

Comment:
Why not simple `git status` ?

Reply:
`--porcelain=v1` ensures backwards compatibility, and after some research, seems to be standard for git scripts.

Reply:
I have never used it, we need simple   behavior if something happened in git repo report it.

Please share link that explains that this is really good option

Reply:
https://stackoverflow.com/a/25149786/13160102
and
https://git-scm.com/docs/git-status#Documentation/git-status.txt---porcelainltversiongt

Reply:
> I have never used it, we need simple behavior if something happened in git repo report it.

@romani the output of the `git status` command is not fixed and can be changed in any future version. Since git is actively used in scripts, the developers added a machine-readable format with versioning. So, "--porcelain=v1 " guarantees that the output of the command will not change in the future. Now there is already version 2, which is slightly different. But the format of version 1 will never change.

Reply:
Please do simple `git status --porcelain` defaults are good for us

Reply:
I revoking my ok to to do non simple git status. Output in Travis is absolutely non for human, we need to do  simple `git status` for check and for printing to log to let user understand it easily.